packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/handlers/DropNodeEventHandler.java

```diff
@@ -131,6 +133,7 @@ private boolean invokeDropNodeTool(IEditingContext editingContext, IDiagramConte
 
             VariableManager variableManager = new VariableManager();
             variableManager.put(Environment.ENVIRONMENT, new Environment(Environment.SIRIUS_COMPONENTS));
+            variableManager.put(IDiagramService.DIAGRAM_SERVICE, new DiagramService(diagramContext));
```

Comment:
Are you sure that it should be `new  DiagramService` here? Why not `new  DiagramServices`?

Reply:
Same remark for every other similar cases in other files.

Reply:
Yes, we want to register an `IDiagramService` variable, on which services from the `IDiagramServices` class can be called (the first parameter of `IDiagramServices` methods is `IDiagramService`).
The names could be reworked though to avoid confusion.